packages/runtime/test/transport/relationships.test.ts

```diff
@@ -942,3 +948,111 @@ describe("Relationship existence check", () => {
+                    {
+                        "@type": "TestRequestItem",
+                        mustBeAccepted: false
+                    }
+                ]
+            },
+            peer: services2.address
+        };
+        const rRequestMessage = await exchangeMessageWithRequest(services3, services2, requestContent);
+        const result = await services2.consumption.incomingRequests.received({
+            receivedRequest: rRequestMessage.content,
+            requestSourceId: rRequestMessage.id
+        });
+
+        expect(result).toBeSuccessful();
+        const requestIds = rRequestMessage.content.id!;
+        const canAcceptResult = (await services2.consumption.incomingRequests.canAccept({ requestId: requestIds, items: [{ accept: true }] })).value;
+        expect(canAcceptResult.isSuccess).toBe(true);
+        await services3.transport.identityDeletionProcesses.initiateIdentityDeletionProcess();
+        await syncUntilHasEvent(services2, PeerToBeDeletedEvent, (e) => e.data.id === relationshipId2);
+        await services2.eventBus.waitForRunningEventHandlers();
+        const updatedRelationship2 = (await services2.transport.relationships.getRelationship({ id: relationshipId2 })).value;
+        expect(updatedRelationship2.peerDeletionInfo?.deletionStatus).toBe("ToBeDeleted");
+
+        const canAcceptResultAfterPeerDeletion = (await services2.consumption.incomingRequests.canAccept({ requestId: requestIds, items: [{ accept: true }] })).value;
+        expect(canAcceptResultAfterPeerDeletion.isSuccess).toBe(false);
+        expect(canAcceptResultAfterPeerDeletion.code).toBe("error.consumption.requests.peerInDeletion");
+    });
+
+    test("messages with multiple recipients should fail if one of the recipients has an active IdentityDeletionProcess", async () => {
+        const result = await failToSendMessageToMultipleRecipients(services2.transport, [services4.address, services1.address]);
+        expect(result).toBeAnError(
+            `The recipient with the address '${services1.address.toString()}' has an active IdentityDeletionProcess so you cannot send a message to him.`,
+            "error.transport.messages.peerInDeletion"
+        );
+    });
+
+    test("messages with multiple recipients should fail with another ErrorMessage if more than one of the recipients has an active IdentityDeletionProcess", async () => {
+        const result = await failToSendMessageToMultipleRecipients(services2.transport, [services3.address, services1.address]);
+        expect(result).toBeAnError(
+            `The recipients with the following addresses '${services3.address.toString()},${services1.address.toString()}' have an active IdentityDeletionProcess so you cannot send a message to them.`,
+            "error.transport.messages.peerInDeletion"
+        );
+    });
+
+    test("sending Notification to an Identity which is in status 'ToBeDeleted' results in an error redirected from the Backbone", async () => {
```

Comment:
In the long term, the backbone should not be treating this an error, this test can go

Reply:
actually the backbone will not block any messages anymore. But yes, I second that this test can be removed as we don't want to test backbone functionalities here. Instead we actually should add a test that shows that you CAN send messages while the other side is marked as ToBeDeleted.

You can write this test and then mark it with `test.skip("...`